HACKING.md

```diff
@@ -154,6 +154,22 @@ run individual tests via `pytest -k`, like so:
 
     pytest -k test-fsinfo.html
 
+You can see JavaScript code coverage information for QUnit tests.  For a
+summary table:
```

Comment:
I just tried this.

It turns out that if you just `./build.js` then the default *is* development mode.  But if you go through the normal build system you end up with production mode.

If we can't open the `.map` file for each reported `.js` file then we simply skip it, so the result is that we end up with no coverage:

```
================================= JavaScript coverage summary =================================
Path                                                     % (bytes covered) ~ uncovered lines
No files considered for coverage

```

I wonder if it would make sense to enable sourcemap output for production builds?  It's also noteworthy that the options for the qunit tests in build.js are separate from those for the rest of the build, so we could always have map files for qunit tests, if we wanted.

Reply:
We already unconditionally disable minification for qunit, so let's just unconditionally enable sourcemaps there, too.

Reply:
Thanks for that update!